### DISCUSSION TITLE
Outside days still show.

### DISCUSSION THREAD

**Discussion by bvelasquez** [gpbl/react-day-picker#1273] [Support]
**Describe the bug**
When showOutsideDays is false, outside days still display when using modifierStyles.

```
<DayPicker
    numberOfMonths={3}
    showOutsideDays={false}
    enableOutsideDaysClick={false}
    disabledDays={{ after: new Date() }}
    selectedDays={selectedDay}
    modifiers={calendarModifiers}
    modifiersStyles={modifierStyles}
    initialMonth={moment().subtract(2, "months").toDate()}
    onDayClick={(day, modifiers = {}) => {
      //
      if (modifiers.disabled) {
        return;
      }
      setSelectedDay(day);
    }}
  />
```

**To Reproduce**
1. Set showOutsideDays to false.
2. Pass modifierStyles, selecting dates that cross over two months.
3. The modiferStyles will show up on outside days.

**Expected behavior**
Nothing should show up when outside days is disabled.

**Screenshots**
![2021-09-13_17-44-06](https://user-images.githubusercontent.com/2302210/133176115-f7e952b3-7d94-4248-8c6d-da7630999a55.png)

Code sandbox example of problem: https://codesandbox.io/s/react-day-picker-examples-selected-forked-0ypgp?file=/example.js

**Additional context**

**Comment by gpbl** [comment]
Yeah i think you need to target those with some  `:not(.outside)`...

**Comment by cgrochowina** [comment]
@bvelasquez hi, ive faced same issue. Overiding modifiers styles with 'outside' key name worked for me. 

example: 

modifiersStyles: {
customDate: {
backgroundColor: 'red' 
},
outside: {
backgroundColor: 'transparent'
}}

Hope it will work for you ;)
